quirks/password-rules.json

```diff
@@ -782,6 +782,9 @@
     "vetsfirstchoice.com": {
         "password-rules": "minlength: 8; required: lower; required: upper; required: digit; allowed: [?!@$%^+=&];"
     },
+    "vince.com": {
+        "password-rules": "minlength: 8; required: digit, lower, upper, [$%/()[]{}=?!.,_*|+~#];"
```

Comment:
What your rules says:
Required: one of a { digit, lower, upper, or special character from this set }

What the site says:

> Required: 8 Characters
> One capital and 1 lowercase letter
> At least 1 number
> At least one special character $%/()[]{}=?!.,-_*|+~#

Which translates to pseudo-code:

Min-length: 8
Required: digit
Required: lower
Required: upper
Required: one of these: $%/()[]{}=?!.,-_*|+~#

Also, `]` is treated specially in our format. It has to be the *last* character of a character class.

With all of that, I think your rule should be:

```
minlength: 8; required: digit; required: lower; required: upper; required: [$%/(){}=?!.,_*|+~#[]];
```

Reply:
@rmondello Please do not be fooled by what the site visually claims to support. Their JS validates as follows: {length: /\S{8,}/, loweruppercase: /(?=.*[A-Z])(?=.*[a-z])/, number: /\d/, special: /[$%/()\[\]{}=?!.,_*|+~#]/}

Please do not put back in the "-" as it won't work.